scripts/table_updates/tests/test_utilities.py

```diff
@@ -0,0 +1,67 @@
+from unittest.mock import MagicMock, create_autospec, patch
+
+import numpy as np
+import pandas as pd
+import pytest
+import synapseclient
+from synapseclient import Schema, Table
+from table_updates import utilities
+
+
+@pytest.fixture(scope="session")
```

Comment:
@rxu17 tweak the test to address your comments and add test case

Reply:
nit: linting with `black` might be helpful for readability

Reply:
Done.